packages/mui-material/src/Typography/Typography.test.js

```diff
@@ -122,4 +122,13 @@ describe('<Typography />', () => {
       marginBottom: '16px',
     });
   });
+
+  it('should check for invalid color value', () => {
+    const { container, debug } = render(
+      <Typography variant="h6" color="background">
+        Hello
+      </Typography>,
+    );
+    debug(container);
```

Comment:
You can add a real test here, like checking whether console.error was called or not

Reply:
I have added the test for console.error using sinon.